sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/ClauseBase.cs

```diff
@@ -1,17 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Azure.DigitalTwins.Core.QueryBuilder
 {
     /// <summary>
     /// Clause objects make up an entire query. There are four different kinds of clauses, SELECT, FROM, JOIN and WHERE.
     /// Each kind of clause takes in a different set of arguments and are implemented as children of a general clause class.
     /// </summary>
-    internal abstract class BaseClause
+    internal abstract class ClauseBase
     {
         /// <summary>
         /// The type of the given clause (out of SELECT, FROM, JOIN and WHERE).
```

Comment:
same here, there is an enum, let the enum dictate what the different options are ...